### ISSUE TITLE
teaser usability of pymovements datasets

### ISSUE THREAD

**Issue by SiQube** [aeye-lab/pymovements#637]
I think it would be cool to teaser one of our use cases (easy accessiblity of datasets) more prominently on the README.md, e.g. adding something like:

Easily access public eye movements datasets in three lines of code:
```python
import pymovmements as pm

dataset = pm.Dataset('JuDo1000', path='data/ToyDataset')
dataset.download(remove_finished=True)
dataset.extract(remove_finished=True)
```

**Comment by dkrako** [comment]
Good idea! We can simplify this example even more by focusing on the most prominent usecase:

```python
import pymovmements as pm

dataset = pm.Dataset('JuDo1000', path='data/judo100')
dataset.download()  # download your dataset
dataset.load()  # load dataset into working memory
```

We can also add two more short example blocks:

```python
dataset.pixdeg()  # transform pixel coordinates to degrees of visual angle
dataset.pos2vel()  # transform positional data to velocity data
```

```python
dataset.detect('ivt')  # detect fixation using the I-VT algorithm
dataset.detect('microsaccades')  # detect saccades using the microsaccades algorithm
```

**Comment by SiQube** [comment]
resolved via #639
